types/npm-cache-filename/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for npm-cache-filename 1.0
+// Project: https://github.com/npm/npm-cache-filename
+// Definitions by: Florian Keller <https://github.com/ffflorian>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function cf(root: string): typeof cf;
```

Comment:
Return type here cannot be `typeof cf` since it just allows you to be called with one parameter so this signature needs to be 
```ts
declare function cf(root: string): (url: string) => string;
declare function cf(root: string, url: string): string;
```